.github/workflows/release.yaml

```diff
@@ -7,43 +7,42 @@ concurrency:
 jobs:
   release:
     if: (github.event_name == 'create' && github.event.ref_type == 'tag') || github.event_name == 'merge_group'
-    name: Release
+    name: Release ${{ matrix.toolchain }} ${{ matrix.platform.target }}
+    strategy:
+      fail-fast: false
```

Comment:
Why do we need to disable fail-fast?

Reply:
Setting fail-fast to false prevents GitHub from cancelling all in-progress jobs if any matrix job fails. This is useful in case build on one of the matrix platforms fails, GitHub will still continue builds on other platforms.